MAUI/Barcode-Generator/migration.md

```diff
@@ -9,9 +9,9 @@ documentation: ug
 
 # Migrate from Xamarin.Forms SfBarcode to .NET MAUI SfBarcodeGenerator
 
-To migrate easier from [Xamarin SfBarcode](https://www.syncfusion.com/xamarin-ui-controls/xamarin-barcode) to [.NET MAUI SfBarcodeGenerator](https://www.syncfusion.com/maui-controls/maui-barcodes), we kept most of the APIs from Xamarin SfBarcode in MAUI SfBarcodeGenerator. However, to maintain the consistency of API naming in MAUI SfBarcodeGenerator, we renamed some of the APIs. The APIs that have been changed in MAUI SfBarcodeGenerator from Xamarin SfBarcode are detailed as follows.
+To make the migration from the [Xamarin SfBarcode](https://www.syncfusion.com/xamarin-ui-controls/xamarin-barcode) to [.NET MAUI SfBarcodeGenerator](https://www.syncfusion.com/maui-controls/maui-barcodes) easier, we kept most of the APIs from the Xamarin SfCircularGauge in the.NET MAUI SfRadialGauge. However, to maintain the consistency of API naming in the.NET MAUI SfRadialGauge, we renamed some of the APIs. Please find the difference in the below topics.
```

Comment:
@SVRagul  Use barcode controls name here.

Reply:
Used the respective control name.